src/NuGet.Services.AzureSearch/AzureSearchScoringConfiguration.cs

```diff
@@ -23,10 +16,10 @@ public class AzureSearchScoringConfiguration
         public Dictionary<string, double> FieldWeights { get; set; }
 
         /// <summary>
-        /// The <see cref="SearchDocument.Full.LogOfDownloadCount"/> magnitude boost.
+        /// The <see cref="SearchDocument.Full.DownloadScore"/> magnitude boost.
         /// This boosts packages with many downloads.
         /// </summary>
-        public double LogOfDownloadCountMagnitudeBoost { get; set; }
+        public double DownloadScoreBoost { get; set; }
```

Comment:
whats the configured value here?

Reply:
`3000`, as per [this file](https://nuget.visualstudio.com/NuGetMicrosoft/_git/NuGetDeployment/pullrequest/974?_a=files&path=%2Fsrc%2FJobs%2FNuGet.Jobs.Cloud%2FJobs%2FDb2AzureSearch%2FPROD%2Fnorthcentralus%2Fappsettings.json)